### ISSUE TITLE
No option to unlock from Accounts tab

### ISSUE THREAD

**Issue by joelamouche** [polkadot-js/apps#4809]
@jacogr 
After a proposal expired, the  "Clear expired democracy locks" doesn't display.
This is because api.tx.utility is undefined in line 70 of Account.tsx for the createClearDemocracyTx function

cc @albertov19

**Comment by jacogr** [comment]
Yes, it doesn't support chains without utility. The effort to support lesser-featured chains just adds maintenance overhead since all needs to be done on-by-one. 

So this is by design.

**Comment by crystalin** [comment]
Thank you @jacogr .
In this case it is using `batch` which I agree is a fair requirement for a chain to have support for democracy, staking and ...

**Comment by jacogr** [comment]
I understand it is crappy, and in a perfect world would love to support everything. In reality, having to support all the operations in optimized batch mode as well as individual mode -

- leads to implementing the same functionality twice
- requires a lot more effort on the unoptimized path (UI-wise, code-wise)
- testing path now gets doubled-up
- ... all code needs maintenance and updates (and in multiple cases this actually means supporting both old and new ways of making calls - in this case the effort just skyrockets)

I'm really not suggesting that all chains go down adding `utility` to their runtime, hence the UI just disables certain functionality where it is not available. In some cases the UI does indeed double-up, for instance in staking you have non-batch "+ stash" operations and batched "+ nominator/+ validator" with a much better UX.

Overall, not having full/equal support for everything is not a position I like being in, but atm, the reverse is not something that can properly be supported at a proper level. If boredom does creep in with less constant changes and functionality at a rapid rate, would certainly like to expand.

TL:DR Would love to... eventually. but honestly with the amount of time it does pop up, the negatives are still way too high.
